src/plugins/nap_monitoring.go

```diff
@@ -55,7 +55,7 @@ func NewNAPMonitoring(env core.Environment, cfg *config.Config) (*NAPMonitoring,
 		log.Warnf("NAP Monitoring report count must be between %v and %v. Defaulting to %v",
 			minReportCountDelimiter,
 			maxReportCountDelimiter,
-			config.Defaults.NAPMonitoring.ReportInterval)
+			config.Defaults.NAPMonitoring.ReportCount)
```

Comment:
Good catch! Probably should add a unit test for things like this to catch and validate sooner.

Reply:
+1 on adding unit tests